src/components/TodoError.tsx

```diff
@@ -0,0 +1,30 @@
+import { useCallback, useContext } from 'react';
+import classNames from 'classnames';
+
+import { TodoContext } from './TodoContext';
+
+export const TodoError = () => {
+  const { state, dispatch } = useContext(TodoContext);
+
+  const hiddenError = useCallback(() => {
```

Comment:
```suggestion
  const hideError = useCallback(() => {
```
function name should be with verb
hiddenError sounds like some error (string/DOM element, etc., which is not visible)